src/main/resources/validationrules/iac/iac-caseworker-mapping.drl

```diff
@@ -53,3 +59,49 @@ then
       .attributes(attr)
       .build());
 end;
+
+rule "iac_senior_tribunal_caseworker_new"
+when
+  $f:  FeatureFlag(status && flagName == FeatureFlagEnum.getIAC_1_1.getValue())
+  $up: UserAccessProfile(roleId == "1", serviceCode == "BFA2", !suspended)
+then
```

Comment:
we can keep both the rules as exactly same and update it in the local for testing

Reply:
done